File: Code Chef/JavaScript/300-399/328 Rain in Chefland RAINFALL1.js
process.stdin.setEncoding("utf-8");

const strToInt = (strArray) => {
    const intArray = strArray.map(Number);
    return intArray;
};

const categorizeRainfall = (rateOfRainfall) => {
    (rateOfRainfall < 3)
        ? console.log("LIGHT")
        : ((rateOfRainfall >= 3) && (rateOfRainfall < 7))
            ? console.log("MODERATE")
            : console.log("HEAVY");
};

process.stdin.on("data", (input) => {
    const testcases = strToInt(input.trim().split("\n").slice(1));
    
    testcases.forEach((rateOfRainfall) => {
        categorizeRainfall(rateOfRainfall);
    });
});

File: Code Chef/C Programming/300-399/328 Rain in Chefland RAINFALL1.c
#include <stdio.h>
int main(void)
{
	int t,x;
	scanf("%d",&t);
	for(int i=0;i<t;i++)
	{
	    scanf("%d",&x);
	    if(x<3)
	    {
	        printf("LIGHT\n");
	    }
	    else if(x>=3 && x<7)
	    {
	        printf("MODERATE\n");
	    }
	    else if(x>=7)
	    {
	        printf("HEAVY\n");
	    }
	}
	return 0;
}